docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_dimmer.htm

```diff
@@ -0,0 +1,225 @@
+
+	</ol>
+
+	<h2><a id="default-selectors-dimmer-wearable"></a>Default Selectors</h2>
+
+	<p>Elements with the <code>class=&quot;ui-dimmer&quot;</code> attribute can create a dimmer.<br>
+
+		<h2><a id="html-examples-dimmer-wearable"></a>HTML Examples</h2>
+
+		<p><a id='create element-dimmer-wearable'></a>
+
+			To create a dimmer using <code>class</code> attribute, use the following code:</p>
+		<pre class="prettyprint">
+    &lt;div class=&quot;ui-dimmer&quot;&gt;&lt/div&gt;
+</pre>
+
+		</li>
+
+		</ul>
+
+		<h2><a id="options-for widget-dimmer-wearable"></a>Options</h2>
+
+		<h3>Summary</h3>
+
+		<table>
+			<tbody>
+				<tr>
+					<th>Option</th>
+					<th>Input type</th>
+					<th>Default value</th>
+					<th>Description</th>
+				</tr>
+
+
+				<tr>
+					<td class="option"><span style="font-family: Courier New,Courier,monospace">value</span></td>
+					<td class="option">number</td>
+					<td class="option">50</td>
+					<td class="option">A currently displayed value by widget.</td>
+				</tr>
+
+				<tr>
+					<td class="option"><span style="font-family: Courier New,Courier,monospace">min</span></td>
+					<td class="option">number</td>
+					<td class="option">0</td>
+					<td class="option">Minimum value that can be displayed by widget.</td>
+				</tr>
+
+				<tr>
+					<td class="option"><span style="font-family: Courier New,Courier,monospace">max</span></td>
+					<td class="option">number</td>
+					<td class="option">100</td>
+					<td class="option">Maximum value that can be displayed by widget.</td>
+				</tr>
+			</tbody>
+		</table>
+
+
+
+		<h2><a id="methods-dimmer-wearable"></a>Methods</h2>
+
+		<p>To call a method on the component, use one of the existing APIs:</p>
+
+		<p>HTML code:</p>
+		<pre class="prettyprint">
+&lt;div class=&quot;ui-dimmer&quot;&gt;&lt/div&gt;
+</pre>
+		<p>JS Code:</p>
+		<pre class="prettyprint">
+var dimmerElement = document.getElementsByClassName(&quot;ui-dimmer&quot;),
+	dimmer = tau.widget.dimmer(dimmerElement);
+
+dimmer.value(40);
+</pre>
+
+		<h3>Summary</h3>
+		<table class="informaltable">
+			<thead>
+				<tr>
+					<th>Method</th>
+					<th>Description</th>
+				</tr>
+			</thead>
+			<tbody>
+				<tr>
+					<td>
+						<pre class="intable prettyprint">number <a href="#method-value">value</a>(number? value) </pre>
+					</td>
+					<td>
+						<p>Gets or sets a value.</p>
+					</td>
+				</tr>
+			</tbody>
+		</table>
+		<dl>
+
+			<dt class="method" id="addidp28072"><code><b><span class="methodName"
+																   id="method-value">value</span></b></code></dt>
+			<dd>
+				<div class="brief">
+					<p>Gets or sets a value.</p>
+				</div>
+				<div class="synopsis">
+					<pre class="signature prettyprint">number value(number? value) </pre>
+				</div>
+
+				<p><span class="version">Since: </span>
+					5.5
+				</p>
+
+				<div class="description">
+					<p>
+						<p>The method returns the current value current value displayed by widget or sets new one if
```

Comment:
you repeat "current value"

Reply:
I have changed it in mobile_Dimmer too. Thanks